lib/kernel.coffee

```diff
@@ -15,7 +15,7 @@ class Kernel
         console.log 'Kernel spec:', @kernelSpec
         console.log 'Kernel configuration:', @config
         console.log 'Kernel configuration file path:', @configPath
-        @language = @kernelSpec.grammarLanguage
+        @language = KernelManager::getLanguageFor @grammar
```

Comment:
Oh I'm sorry I didn't restart atom while testing, so this error slipped through.

`Kernel::grammar` is used in this [line](https://github.com/nteract/hydrogen/blob/079608feca61a9c419cbcc4445cfc68252a271d7/lib/watch-view.coffee#L16). I wouldn't remove it.

I think we should switch back to `@language = @kernelSpec.language`